packages/L10n/src/locales-js/en.js

```diff
@@ -73,7 +73,7 @@ const locales = {
         "arrowdown": "down",
         "escape": "escape"
       },
-      "externaLink": {
+      "externalLink": {
```

Comment:
After running `yarn` The en.js file seems to have updated with the correct key "enternalLins", but not the other .js files for other languages. Unsure if i should fix the other typos in other .yml files manually at this point 🤷‍♂

Reply:
Only if it can't wait until next Thursday when we'll check back in next batch. @tristanjasper